wheel/wheelfile.py

```diff
@@ -19,6 +19,20 @@
     re.VERBOSE)
 
 
+# `hashlib.algorithms_available` was added in Python 2.7.9 / 3.2.
+if hasattr(hashlib, 'algorithms_available'):
+    def _hashlib_has_algorithm(algorithm):
```

Comment:
Could we rather not have the attribute check inside the function itself, instead of defining two functions?
Any particular benefit of this?